cl_sii/libs/xml_utils.py

```diff
@@ -478,12 +480,17 @@ def verify_xml_signature(
         #
         #   Source:
         #   https://github.com/XML-Security/signxml/commit/ef15da8dbb904f1dedfdd210ae3e0df5da535612
-        result: signxml.VerifyResult = xml_verifier.verify(
+        result = xml_verifier.verify(
             data=tmp_bytes,
             require_x509=True,
             x509_cert=trusted_x509_cert_open_ssl,
             ignore_ambiguous_key_info=True,
+            expect_config=signxml.verifier.SignatureConfiguration(
+                signature_methods=frozenset([signxml.algorithms.SignatureMethod.RSA_SHA1]),
+                digest_algorithms=frozenset([signxml.algorithms.DigestAlgorithm.SHA1]),
+            ),
```

Comment:
This may be a breaking change. Verify that and update dependency constraints if necessary: https://github.com/fyntex/lib-cl-sii-python/blob/b669efd2f47dad726379e763495bd7c25a1f5e95/setup.py#L32

Reply:
Fixed in https://github.com/fyntex/lib-cl-sii-python/pull/462.